### PR TITLE
Sequence_empty ONNX function

### PR DESCRIPTION
**Pull request by Fayad-Alman**
- Added ONNX function Sequence Empty to Ivy API
- Updated Array API

### CODE REVIEW

**Review by ZoeCD — CHANGES_REQUESTED** [review]
LGTM! Now you need to add the test function! Let me know if you have any questions

**Comment by Fayad-Alman** [comment]
@ZoeCD Force pushed master to compensate for conflicts. Will be updating soon!

**Comment by Fayad-Alman** [comment]
@ZoeCD 
Done! Mind taking a look?

**Comment by Fayad-Alman** [comment]
Note: Most (if not all) failures are resulting from other files

The creation failures originate from a function triu_indices()

File: ivy/functional/ivy/experimental/creation.py [hunk 1]
```diff
@@ -575,3 +575,49 @@ def eye_like(
...
+        an array having the same shape as x and containing uninitialized data.
+    This function conforms to the `Array API Standard
+    <https://data-apis.org/array-api/latest/>`_. This docstring is an extension of the
+    `docstring <https://data-apis.org/array-api/latest/API_specification/generated/signatures.creation_functions.empty_like.html>`_ # noqa
+    in the standard.
+    Both the description and the type hints above assumes an array input for simplicity,
+    but this function is *nestable*, and therefore also accepts :class:`ivy.Container`
+    instances in place of any of the arguments.
+    """
+    return current_backend(x).empty_like(shape, dtype=dtype, device=device, out=out)
```
Comment (ZoeCD) [inline ivy/functional/ivy/experimental/creation.py:623]: Just a tiny thing. You're calling `empty_like` not `sequence_empty`

**Review by ZoeCD — CHANGES_REQUESTED** [review]

File: ivy/functional/ivy/experimental/creation.py (hunk 1 above)
Comment (Fayad-Alman) [reply ivy/functional/ivy/experimental/creation.py]: Sorry about that! Should be fixed now.

File: ivy/array/experimental/creation.py [hunk 2]
```diff
@@ -54,3 +54,38 @@ def eye_like(
                     0., 1., 0.]])
         """
         return ivy.eye_like(self._data, k=k, dtype=dtype, device=device, out=out)
+    
+    
+    def sequence_empty(
```
Comment (ZoeCD) [inline ivy/array/experimental/creation.py:59]: Sorry! Just one more thing. I noticed there are some lint errors related to your changes. Here they are: 
```
ivy/array/experimental/creation.py:59:5: E303 too many blank lines (2)
ivy/array/experimental/creation.py:6:89: E501 line too long (89 > 88 characters)
ivy/container/experimental/creation.py:819:5: E303 too many blank lines (2)
ivy/container/experimental/creation.py:845:5: E303 too many blank lines (2)
ivy/container/experimental/creation.py:867:1: W391 blank line at end of file
```
Don't worry about the others. Did you install pre-commit?

File: ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py [hunk 3]
```diff
@@ -301,3 +301,42 @@ def test_eye_like(
...
+    *,
+    shape,
+    dtype,
+    test_flags,
+    backend_fw,
+    fn_name,
+    on_device,
+    ground_truth_backend,
+):
+    ret = helpers.test_function(
```
Comment (ZoeCD) [inline ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py:330]: Oh, and here
`ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py:330:5: F841 local variable 'ret' is assigned to but never used`

File: ivy/functional/ivy/experimental/creation.py [hunk 4]
```diff
@@ -575,3 +575,50 @@ def eye_like(
...
+    
+    This function conforms to the `Array API Standard
+    <https://data-apis.org/array-api/latest/>`_. This docstring is an extension of the
+    `docstring <https://data-apis.org/array-api/latest/API_specification/generated/signatures.creation_functions.empty_like.html>`_ # noqa
+    in the standard.
+    Both the description and the type hints above assumes an array input for simplicity,
+    but this function is *nestable*, and therefore also accepts :class:`ivy.Container`
+    instances in place of any of the arguments.
+    """
+    return current_backend(x).sequence_empty(shape, dtype=dtype, device=device, out=out)
```
Comment (ZoeCD) [inline ivy/functional/ivy/experimental/creation.py:624]: `ivy/functional/ivy/experimental/creation.py:624:28: F821 undefined name 'x'`
No need to send `x`. Nonetheless, please check that this doesn't cause the test to fail

**Review by ZoeCD — CHANGES_REQUESTED** [review]

**Comment by Fayad-Alman** [comment]
Hi @ZoeCD 
After multiple checks, I realized that we pretty much have the Sequence empty function implemented in Ivy
Feel free to resolve the issue attached to this PR
I'll be closing it with this comment
Apologies for dragging you through this unnecessary mess

**Pull request closed without merging**